rise/static/main.js

```diff
@@ -931,10 +933,24 @@ define([
                      },
                      "edit-all-cells", "RISE");
 
+    // because the `Edit Keyboard Shortcuts` utility does not mention the
+    // actions prefix (i.e. 'RISE' in our case), we choose to make these two
+    // action names start with `rise-` even if it's a bit redundant.
+
+    // define an action that goes to the nbconfigurator page for rise
+    let nbconfigurator = function() {
+      let url = "/nbextensions/?nbextension=rise/main";
+      window.open(url, '_blank');
+    }
+
+    actions.register({ help: 'open the nbconfigurator page for RISE',
+                       handler: nbconfigurator},
+                     "rise-nbconfigurator", "RISE");
+
     // mostly for debug / information
-    actions.register({ help   : 'show RISE config in console',
+    actions.register({ help   : 'output RISE configuration in console, for debugging mostly',
                        handler: showConfig},
-                     "show-config", "RISE");
+                     "rise-dump-config", "RISE");
```

Comment:
`showConfig` is commented in the [source code](https://github.com/parmentelat/RISE/blob/master/rise/static/main.js#L1033), maybe we can make it an option through config as well? Or do you want to keep it hidden?

Reply:
yeah, show-config really is something that end users won't care about at all, it just dumps the config struct in the javascript console..